sdcm/nemesis.py

```diff
@@ -2670,11 +2670,8 @@ def decommission_nodes(self, add_nodes_number, rack, is_seed: Optional[Union[boo
             else:
                 self.set_target_node(is_seed=is_seed)
             self.log.info("Next node will be removed %s", self.target_node)
-            try:
-                InfoEvent(message='StartEvent - ShrinkCluster started decommissioning a node').publish()
```

Comment:
This is not good, we need this for the latency calculations in performance tests.
+ it's much clearer than the nodetool event.

Reply:
fixed